requirements/dev.txt

```diff
@@ -25,7 +25,7 @@ cffi==1.15.0
     # via
     #   -r requirements/quality.txt
     #   pynacl
-chardet==4.0.0
+chardet==5.0.0
```

Comment:
No specific breaking change https://github.com/chardet/chardet/releases/tag/5.0.0 related to our code usage.